### PR TITLE
man/options.rst: fix typo, layout

### PR DESCRIPTION
**Pull request by fhlfibh**
I agree that my changes can be relicensed to LGPL 2.1 or later.

### CODE REVIEW

File: DOCS/man/options.rst [hunk 1]
```diff
@@ -4338,11 +4338,7 @@ The following video options are currently all specific to ``--vo=opengl`` and
     v-log
         Panasonic V-Log (VARICAM) curve
 
-    NOTE: When using HDR output formats, mpv will encode to the specified
-          curve but it will not set any HDMI flags or other signalling that
-          might be required for the target device to correctly display the
-          HDR signal. The user should independently guarantee this before
-          using these signal formats for display.
+    NOTE: When using HDR output formats, mpv will encode to the specified curve but it will not set any HDMI flags or other signalling that might be required for the target device to correctly display the HDR signal. The user should independently guarantee this before using these signal formats for display.
```
Comment (wiiaboo) [inline DOCS/man/options.rst]: This isn't readable in a text editor without wrapping.
Just remove the extra indentation.

**Comment by wm4** [comment]
You also want to use a note section, look for instances of `.. note::`.

**Comment by wm4** [comment]
Still not right...

**Comment by wm4** [comment]
Squashed the commits, added a new line after the note (it's unclear whether RST requires this, but all other uses have it), and pushed.

**Comment by fhlfibh** [comment]
Manipulating github things seems a bit hard for me...

**Pull request closed without merging**